packages/terra-table/src/Table.jsx

```diff
@@ -37,33 +32,27 @@ const defaultProps = {
 
 const Table = ({
   children,
+  header,
   isStriped,
   isPadded,
   ...customProps
 }) => {
   const tableClassNames = cx([
     'table',
-    { striped: isStriped },
-    { padded: isPadded },
     customProps.className,
   ]);
+
   return (
-    <table {...customProps} className={tableClassNames}>
-      {children}
+    <table {...customProps} data-table-padded={isPadded} className={tableClassNames}>
+      {header}
+      <tbody data-table-striped={isStriped} className={cx(['body'])}>
+        {children}
+      </tbody>
     </table>
   );
 };
 
 Table.propTypes = propTypes;
 Table.defaultProps = defaultProps;
-Table.Rows = TableRows;
```

Comment:
Should we create an `index.js` file that will export each Table component and then update the main file in the package.json to be `index.js`?

Reply:
never mind! I got far enough to see you already have it.